lib/galaxy/dependencies/pipfiles/default/Pipfile

```diff
@@ -70,7 +70,7 @@ bioblend = "*"
 boto = "*"
 kombu = "*"
 psutil = "*"
-pulsar-galaxy-lib = ">=0.14.0.dev1"
+pulsar-galaxy-lib = "==0.14.0.dev1"
```

Comment:
This is necessary because of a pipenv issue: https://github.com/pypa/pipenv/issues/3928